packages/workflow/src/Interfaces.ts

```diff
@@ -561,6 +561,30 @@ namespace ExecuteFunctions {
 			| 'resolveData';
 	}
 
+	namespace StringReturning {
+		export type NodeParameter =
+			| 'binaryProperty'
```

Comment:
can we limit this to `binaryProperty`, `operation`, and `resource` for now?, as properties others are specific to nodes, and it might make sense to eventually type these on a per node basis.
i.e. `query` is a string in most places, but if a node wants to define it as a `ParsedQs` from `qs`, they won't be able to do that.

Reply:
Are you sure? I was planning to allow specifying non-overload types with generics.